internal/cli/install.go

```diff
@@ -22,6 +22,20 @@ var NeededOptions = kubernetes.InstallationOptions{
 		Default:     "epinio@suse.com",
 		Value:       "",
 	},
+	{
+		Name:        "user",
+		Description: "The user name for authenticating all API requests",
+		Type:        kubernetes.StringType,
+		Default:     "me",
+		Value:       "",
+	},
+	{
+		Name:        "password",
+		Description: "The password for authenticating all API requests",
+		Type:        kubernetes.StringType,
+		Default:     "change",
```

Comment:
I would prefer a "secure by default" approach. We can generate a password when the user doesn't provide one instead of using a hardcoded one. The user may forget to provide a password and given we don't have a way for the user to change it (yet), they will be forced to re-install. Even when changing the password will be possible, it's still better if we generated a good password by default so the user doesn't have to come up with a good password.

We have 2 personas, the cluster admin installing Epinio and the application developer.
The admin needs to have access to the cluster with kubectl/kubeconfig because that's how we create the various resources when installing Epinio.
The app developer, may or may not have kubectl access to the cluster. Right now we still try to access the cluster directly in some places (e.g. when trying to figure out what the domain is) but we should get to the point where it's possible to use Epinio with no direct access to the cluster, unless through Epinio server.

We can then have a group of subcommands (e.g. under the `admin` command) that are meant to be used by the administrator. Those are the commands that need direct access to the cluster.
Examples

- `epinio admin install` (the current `epinio install`)
- `epinio admin uninstall` (the current `epinio uninstall`)
- `epinio admin get-credentials`
- `epinio admin change-credentials`
- etc

With this in mind, I imagine the final process as:

- Admin installs Epinio on the cluster
- After installation is done, a command is printed that the user can run to read the credentials from the kube secret (e.g. `epinio admin get-credentials`). Alternatively, when the admin user is also the application developer, a command like `epinio admin login` can be used that will not require the user to know the credentials. It will simply copy the credentials from the secret into the Epinio config.
- If the application developer is a different user, then the admin will give her the credentials so the app dev doesn't need to have direct access to the cluster. 

This is just a proposal, we need to discuss it a bit. Let's keep things as they are here and improve on another PR. We still have to see if printing and storing credentials in plain text is ok.

Reply:
Let's fix it as part of this: https://github.com/epinio/epinio/issues/61#issuecomment-841161324